cypress/integration/version/task_filters.ts

```diff
@@ -208,28 +209,21 @@ describe("Tasks filters", () => {
             search: "success",
           });
           waitForTable();
-          cy.dataCy("filtered-count")
-            .invoke("text")
-            .should("have.length.greaterThan", 0);
 
-          cy.dataCy("filtered-count")
-            .invoke("text")
-            .then((postFilterCount) => {
-              cy.dataCy("filtered-count").should("have.text", preFilterCount);
-              selectCheckboxOption("Succeeded", false);
-              urlSearchParamsAreUpdated({
-                pathname: pathTasks,
-                paramName: urlParam,
-                search: null,
-              });
-              waitForTable();
-              cy.dataCy("filtered-count").should("have.text", postFilterCount);
-            });
+          cy.dataCy("filtered-count").should("have.text", 0);
```

Comment:
I noticed this as well and this behavior existed before the test data updates I think it may have something to do with the query. It works fine in https://github.com/evergreen-ci/evergreen/pull/6760 so I think something wonky must have been going no with the test data and the old query.